src/sentry/tasks/sentry_apps.py

```diff
@@ -257,6 +256,14 @@ def workflow_notification(installation_id, issue_id, type, user_id, *args, **kwa
     send_webhooks(installation=install, event=f"issue.{type}", data=data, actor=user)
 
 
+@instrumented_task("sentry.tasks.send_process_resource_change_webhook", **TASK_OPTIONS)
+@retry(**RETRY_OPTIONS)
+def send_resource_change_webhook(installation, event, data, *args, **kwargs):
```

Comment:
@NisanthanNanthakumar I don't think we should pass the `installation` as a parameter to the task since this will use pickling. Instead, we should pass an `installation_id` and look up the installation in the task.

Reply:
@scefali hmm that means another db lookup for each task. What are the downsides of using the pickled object? It seems that all the tests are passing with this current implementation.

Reply:
@NisanthanNanthakumar I believe it will make certain Python upgrades challenging. Dan can give you more information if you're really curious. See how we do other tasks: we pass IDs to objects and look up the objects in the task. If we are worried about another DB lookup, we should add caching on the installation.